pyoozie/client.py

```diff
@@ -10,10 +10,6 @@
 from pyoozie import model
 
 
-logger = logging.getLogger('pyoozie.OozieClient')
-logging.getLogger('requests').setLevel(logging.WARNING)
```

Comment:
This is probably useful for debugging but it's an import side-effect and we should let library users define their own requests logging levels.